chain/client/tests/challenges.rs

```diff
@@ -365,11 +365,11 @@ fn challenge(
 #[test]
 fn test_verify_chunk_invalid_state_challenge() {
     let store1 = create_test_store();
-    let genesis = Genesis::test(vec!["test0", "test1"], 1);
+    let genesis = Arc::new(Genesis::test(vec!["test0", "test1"], 1));
     let runtimes: Vec<Arc<dyn RuntimeAdapter>> = vec![Arc::new(neard::NightshadeRuntime::new(
         Path::new("."),
         store1,
-        Arc::new(genesis),
+        genesis.clone(),
```

Comment:
I still prefer explicit `Arc::clone(&genesis)` to clearly understand that the clone is cheap.